rfcs/20200818-adaptive-pooling/pooling.md

```diff
@@ -0,0 +1,160 @@
+%% Proposal to extend the documentation by adaptive pooling. Please, see the Implementation Limitations section %%
+
+
+
+
+Pooling {#dev_guide_pooling}
+============================
+
+>
+> [API Reference](@ref dnnl_api_pooling)
+>
+
+The pooling primitive performs forward or backward max or average pooling
+operation on 1D, 2D, or 3D spatial data.
+
+The pooling operation is defined by the following formulas.
+We show formulas only for 2D spatial data which are straightforward to
+generalize to cases of higher and lower dimensions. Variable names follow the
+standard @ref dev_guide_conventions.
+
+### Forward
+
+Max pooling:
+
+\f[
+    \dst(n, c, oh, ow) =
+        \max\limits_{kh, kw}
+        \left(
+            \src(n, c, oh \cdot SH + kh - PH_L, ow \cdot SW +kw - PW_L)
+        \right)
+\f]
+
+Average pooling:
+
+\f[
```

Comment:
Some formulas are rendered incorrectly. Please fix it.

Reply:
Done.